test/binary-parser.test.js

```diff
@@ -313,7 +313,7 @@ function pathSetBinaryTests() {
      0000585E1F3BD02A15D6185F8BB9B57CC60DEDDB37C13157180C769B66D942EE
      69E6DCC940CA48D82337AD000000000000000000000000425443000000000057
      180C769B66D942EE69E6DCC940CA48D82337AD10000000000000000000000000
-     58525000000000003000000000000000000000000055534400000000000A20B3
+     00000000000000003000000000000000000000000055534400000000000A20B3
```

Comment:
Would it make sense to add a new/duplicate test so that both `"0000000000000000000000005852500000000000"` and `"XRP"` cases are tested for expected behavior?

Reply:
`58525` is disallowed: https://xrpl.org/serialization.html#currency-codes